From f22dabb45448e5a8d2df85703fd72b0f83fe7eda Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Thu, 21 Apr 2016 00:44:04 -0400
Subject: [PATCH] include sys/sysmacros.h for major/minor/makedev

These funcs are defined in the sys/sysmacros.h header, not sys/types.h.
Linux C libraries are updating to drop the implicit include, so we need
to include it explicitly.
---
 filesys.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/filesys.c b/filesys.c
index 2ed25f4..9b59998 100644
--- a/filesys.c
+++ b/filesys.c
@@ -16,6 +16,7 @@
  */
 
 #include "defs.h"
+#include <sys/sysmacros.h>
 #include <linux/major.h>
 #include <regex.h>
 #include <sys/utsname.h>
-- 
2.7.4

